geoips/commandline/geoips_command.py

```diff
@@ -38,11 +37,12 @@ def __init__(self):
         get_plugin_packages() and get_plugin_package_paths() functions.
         """
         self.entrypoints = [
-            ep.value for ep in sorted(get_entry_point_group("geoips.plugin_packages"))
+            ep.value
+            for ep in sorted(metadata.entry_points(group="geoips.plugin_packages"))
```

Comment:
Is this in another PR?

Reply:
Yes, this is in v1.13 release PR however I needed this functionality in this branch otherwise the CLI would not work. This change will be removed once v1.13 is released. If this is merged first, it should not cause conflicts.